app/src/main/java/io/pslab/fragment/ThermometerDataFragment.java

```diff
@@ -437,6 +441,10 @@ private void writeLogToFile(long timestamp, float sensorReading) {
         if (getActivity() != null && thermoSensor.isRecording) {
             if (thermoSensor.writeHeaderToFile) {
                 thermoSensor.csvLogger.prepareLogFile();
+                SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+                String metaDataTime = sdf.format(System.currentTimeMillis());
+                String metaData = getResources().getString(R.string.thermometer) + "," + metaDataTime.split(" ")[0] + "," + metaDataTime.split(" ")[1];
+                thermoSensor.csvLogger.writeCSVFile(metaData);
```

Comment:
I see these 4 lines everywhere. In Java we can create a method with this repetitive code and replace these 4 lines with one line method call and implement the function inside CSVLogger class.